css/deeplinking.scss

```diff
@@ -0,0 +1,73 @@
+.deeplinking {
+    background-color: #fff;
+    width: 100%;
+    height: 100%;
+    .header {
+        width: 100%;
+        height: 70px;
+        background-color: #f1f2f5;
+        .logo {
+            margin-top: 15px;
+            margin-left: 50px;
+            height: 40px;
+            float: left;
+        }
+        .appName {
+            float: left;
+            margin-top: 15px;
+            margin-left: 5px;
+            height: 40px;
+            font-size: 25px;
+        }
+    }
+    .content {
+        top: 110px;
+        bottom: 40px;
+        left: 0px;
+        right: 0px;
+        position: fixed;
```

Comment:
Why do you need this position fixed?

Reply:
You can't set the height or max-height to 40?

Reply:
But the height should be whatever is left of the visible area!